source/_integrations/icloud.markdown

```diff
@@ -3,12 +3,27 @@ title: "iCloud"
 description: "Instructions on how to use iCloud to track devices in Home Assistant."
 logo: icloud.png
 ha_category:
+  - Hub
   - Presence Detection
-ha_release: "0.10"
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_release: 0.10
 ---
 
 The `icloud` integration allows you to detect presence using the [iCloud](https://www.icloud.com/) service. iCloud allows users to track their location on iOS devices.
 
+There is currently support for the following services and platforms within Home Assistant:
+
+- [Platforms](#platforms)
+  - [Device Tracker](#device-tracker)
+  - [Sensor](#sensor)
+- [Services](#services)
```

Comment:
We don't add a TOC to our pages, we do reference supported platforms.
So either limit the list to supported platforms and their links, or remove it entirely.

Reply:
OK, I've randomly seen it from Hive actually.

But while checked, this is the only one, will do a PR to remove it.

Reply:
Here it is : #11443